aws/resource_aws_rds_cluster.go

```diff
@@ -623,9 +623,19 @@ func resourceAwsRDSClusterUpdate(d *schema.ResourceData, meta interface{}) error
 	}
 
 	if requestUpdate {
-		_, err := conn.ModifyDBCluster(req)
+		err := resource.Retry(5*time.Minute, func() *resource.RetryError {
+			_, err := conn.ModifyDBCluster(req)
+			if err != nil {
+				awsErr, ok := err.(awserr.Error)
+				if ok && awsErr.Code() == "InvalidDBClusterStateFault" {
```

Comment:
We don't actually use constants anywhere else in the provider in this context, but it sounds like a good idea.